test/unit/util/audio-util.test.js

```diff
@@ -87,11 +87,15 @@ describe('dropEveryOtherSample', () => {
         sampleRate: 2
     };
     test('result is half the length', () => {
-        const {samples} = dropEveryOtherSample(buffer, 1);
+        const {samples} = dropEveryOtherSample(buffer);
         expect(samples.length).toEqual(Math.floor(buffer.samples.length / 2));
     });
     test('result contains only even-index items', () => {
-        const {samples} = dropEveryOtherSample(buffer, 1);
```

Comment:
These extra args were leftover from a previous incarnation of the function, another thing unit testing didn't show.